### DISCUSSION TITLE
2022/01/为博客支持评论系统/

### DISCUSSION THREAD

**Discussion by giscus** [SouthFox-D/SouthFox-D.github.io#12] [博客评论]
# 2022/01/为博客支持评论系统/

突然发现加入十年之约的博客评论区都好热闹……似乎都是随机访问过来的，所以还是有必要为博客加上评论系统的？

https://blog.southfox.me/2022/01/%E4%B8%BA%E5%8D%9A%E5%AE%A2%E6%94%AF%E6%8C%81%E8%AF%84%E8%AE%BA%E7%B3%BB%E7%BB%9F/

**Comment by SouthFox-D** [comment]
对了，如果不想多授权个应用，也可以直接点击 x 条评论直接到 `GitHub Discussions` 发言，不过没有评论的评论区就不会在 `Discussions` 里创建对应帖子……或许以后有对应解决方案吧。

**Comment by lingyf** [comment]
关注博主好久了，支持一个👍

**Comment by Irithys** [comment]
咕噜！

**Comment by VIMIN-CC** [comment]
主题很有年代感啊
